src/components/Item.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { addFavorite, removeFavorite } from '../actions';
+import { FavButton, ItemDiv } from './styled';
+
+class Item extends Component {
+  handleClick() {
+    const {
+      addFavorite,
+      removeFavorite,
+      isFav,
+      item
+    } = this.props;
+
+    if (isFav) {
+      removeFavorite(item.id);
+    } else {
+      addFavorite(item);
+    }
+  }
+
+  render() {
+    const { item, isFav, imgUrl } = this.props;
+    return (
+      <ItemDiv>
+        <NavLink to={`/Details/${item.id}`}>
```

Comment:
this can also use the [match](https://reacttraining.com/react-router/core/api/match) object to get more information about the current route and have something like `/${match. url}/${item.id}`, having this you can maintain the 'Detail' word at higher levels of configuration, allowing you to change it in one place.
`Link` vs `NavLink` is mostly about the `activeClassName` prop, since you don't really care about the active state here, rather use `Link`